### ISSUE TITLE
cache_status data definition and properties

### ISSUE THREAD

**Issue by andrixnet** [opencaching/opencaching-pl#880]
Cache status defines the state of a cache. Each cache status is identified by a unique ID.

These are specific to owners editing caches and OCTeam for reviewing or blocking caches.

Properties:
- ID
- name (for referencing translations)
- base image icon (overlay ?)
- published (only available after being published; eg available, disabled)
- review (only available before being published; eg ready to be published)
- admin (can be set by OCTeam only)

**Comment by kojoty** [comment]
What should be done here? Can anyone clarify?

**Comment by following5** [comment]
I think this is about OOP and translation refactoring of cache status - move all "knowledge" about cache status into the GeoCache class, and make the `cache_status` table obsolete.

OCUS surely did not change the set of cache statuses, so this should not affect OCUS migration.

**Comment by andrixnet** [comment]
If all usage of `cache_status` table has been removed and placed by proper code, then, it should be safe to drop that table on all nodes, right?

**Comment by following5** [comment]
Right. But I would give higher priority to eliminating `cache_type`, `cache_attrib` and `countries`, which have different content on different nodes.
